client/components/infinite-list/index.jsx

```diff
@@ -66,12 +69,10 @@ export default class InfiniteList extends React.Component {
 		this.scrollHelper = new ScrollHelper( this.boundsForRef );
 		this.scrollHelper.props = this.props;
 		if ( this._contextLoaded() ) {
-			this._scrollContainer = this.props.context || window;
+			this._scrollContainer = this.scrollHelper.props.context || window;
```

Comment:
Is there any difference between `this.props` and `this.scrollHelper.props` here?

Reply:
Not really. Hmm, I'm not sure why I made that change.

This has now been made irrelevant due to the merging of `ScrollHelper` and `InfiniteList`.